recipes/shutilwhich-cwdpatch/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "shutilwhich-cwdpatch" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 500b820b5aaa5f5dda7c7b9151c4950dc0300e42664435207fb19d6e7cbb6217
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - shutilwhich_cwdpatch
+
+about:
+  home: https://github.com/kiwi0fruit/shutilwhich-cwdpatch
+  license: PSFL
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: Python-2.0
```